pkg/scripts/environment_test.go

```diff
@@ -43,30 +41,30 @@ func equalArrayContents(a []string, b []string) bool {
 	return true
 }
 
-func TestConvertEnvironmentToDocker(t *testing.T) {
```

Comment:
This test seems to run a pseudo-Docker build. It pulls in a lot of deps from `docker/docker` which seem to break things.

Moving this out for the time being.